sky/templates/jobs-controller.yaml.j2

```diff
@@ -27,9 +27,7 @@ setup: |
   {% endif %}
 
   # Dashboard.
-  ps aux | grep -v nohup | grep -v grep | grep -- "-m sky.spot.dashboard" | awk '{print $2}' | xargs kill > /dev/null 2>&1 || true
-  pip list | grep flask  > /dev/null 2>&1 || pip install flask 2>&1 > /dev/null
-  ((ps aux | grep -v nohup | grep -v grep | grep -q -- "-m sky.jobs.dashboard.dashboard") || (nohup {{ sky_python_cmd }} -m sky.jobs.dashboard.dashboard >> ~/.sky/job-dashboard.log 2>&1 &));
+  {{ dashboard_setup_cmd }}
```

Comment:
Can we also move this into run commands, so that it will be started by `sky jobs launch --fast`?